docs/bindings/java.md

```diff
@@ -171,7 +171,7 @@ class MyBuildServer extends BuildServer {
   def buildTargetCompile(params: CompileParams): CompletableFuture[CompileResult] = ???
   def buildTargetDependencySources(params: DependencySourcesParams): CompletableFuture[DependencySourcesResult] = ???
   def buildTargetDependencyModules(params: DependencyModulesParams): CompletableFuture[DependencyModulesResult] = ???
-  def buildTargetInverseSources(params: InverseSourcesParams): CompletableFuture[InverseSourcesResult] = ???
```

Comment:
what happened to `buildTargetInverseSources` here?

Reply:
Ops, reverting